plugins/out_http/http.c

```diff
@@ -328,13 +359,179 @@ static int http_gelf(struct flb_out_http *ctx,
         s = tmp;
     }
 
-    ret = http_post(ctx, s, flb_sds_len(s), tag, tag_len);
+    ret = http_post(ctx, s, flb_sds_len(s), tag, tag_len, NULL);
     flb_sds_destroy(s);
     msgpack_unpacked_destroy(&result);
 
     return ret;
 }
 
+static char **extract_headers(mpack_reader_t *reader) {
```

Comment:
out_http uses msgpack-c library. Please migrate the mpack code to msgpack-c

Reply:
OK I will switch.

For the record, the reason I used mpack is that you mentioned it was better for new code since it was faster.